system-addon/lib/HighlightsFeed.jsm

```diff
@@ -55,23 +55,18 @@ this.HighlightsFeed = class HighlightsFeed {
   }
 
   async fetchHighlights(broadcast = false) {
+    // We need TopSites to have been initialised for deduping.
+    // We can return early because we will get another chance to initialize
+    // on TOP_SITES_UPDATED.
```

Comment:
nit: odd wrapping and probably can just be shorter while still clear with the code below it:
`// We need TopSites for deduping, so just wait for TOP_SITES_UPDATED`